packages/backend/package.json

```diff
@@ -112,8 +114,9 @@
     },
     "scripts": {
         "generate-api": "tsoa spec-and-routes --configuration tsoa.yml",
+        "generate-api-dev": "chokidar './src/controllers/**/*.ts' -c 'yarn run generate-api'",
```

Comment:
im not going to introduce this as part of the dev mode (for now)

Reply:
let's see if it's effective and then introduce it to the dev

Reply:
we should also regenerate the api if any underlying model type changes, which is harder to do...